packages/veto-app/frontend/src/test-utils/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { ThemeProvider } from 'emotion-theming'
+import { lightTheme } from '@veriphi/veto-ui'
+
+const AllTheProviders = ({ children }) => {
+  return <ThemeProvider theme={lightTheme}>{children}</ThemeProvider>
+}
+
+const customRender = (ui, options?: any) => render(ui, { wrapper: AllTheProviders, ...options })
```

Comment:
:thinking: TS should be complaining here since `any | undefined` can't always be spread.
If it complains, change the optional flag `?` to a default of `{}`
`options: any = {}`